dora/stdlib/Double.dora

```diff
@@ -4,6 +4,16 @@
   @internal fun toFloat() -> Float;
   @internal fun toString() -> String;
 
+  fun toStringBinary() -> String {
```

Comment:
Couldn't we share all those implementations in a single function (apart from bool)? Might also be a good use-case for the StringBuffer class.

Reply:
Had considered it, but I didn't want to have a "public" method somewhere that was only supposed to be used internally. Where would you place it?

Reply:
I suppose in any of these files is good enough for now. That this function is effectively `public` for now is definitely unfortunate, let's hope we will have some form of encapsulation sooner or later.

Reply:
Alright, I updated the PR.